### PR TITLE
Forms: validate forms with yup

### PR DESCRIPTION
**Pull request by KonstantinaStoikou**
Closes partially #230

### CODE REVIEW

File: src/lib/config/defaultConfig.js [hunk 1]
```diff
@@ -117,7 +117,7 @@ export const RECORDS_CONFIG = {
       sort: [
         {
           order: 1,
-          sortBy: 'mostrecent',
+          sortBy: '-mostrecent',
```
Comment (ntarocco) [inline src/lib/config/defaultConfig.js:120]: I would have opted for something more understandable:
```
sortBy: 'mostrecent',
sortOrder: 'desc',
```
but I am fine with this :)

File: src/lib/pages/backoffice/Series/SeriesForms/SeriesEditor/components/SeriesForm/AccessUrls.js [hunk 2]
```diff
@@ -46,7 +46,7 @@ export class AccessUrls extends Component {
           <ArrayField
             fieldPath="access_urls"
             defaultNewValue={{
-              url: undefined,
+              value: undefined,
```
Comment (ntarocco) [inline src/lib/pages/backoffice/Series/SeriesForms/SeriesEditor/components/SeriesForm/AccessUrls.js:49]: Nicely spotted! Thanks a lot!

**Review by ntarocco — APPROVED** [review]

**Review by ntarocco — COMMENTED** [review]
Any change to dependencies that you have committed ` package-lock.json`?
Travis is also failing.

File: src/lib/config/defaultConfig.js (hunk 1 above)
Comment (KonstantinaStoikou) [reply src/lib/config/defaultConfig.js:120]: Without the minus in front the order is the opposite (less recent)

Comment (ntarocco) [reply src/lib/config/defaultConfig.js:120]: I am quickly checking this point in my local instance

Comment (ntarocco) [reply src/lib/config/defaultConfig.js:120]: As discussed, please remove this change, I will create 2 PRs to fix frontend/backend

**Review by ntarocco — APPROVED** [review]

**Review by kprzerwa — APPROVED** [review]

**Pull request merged**